docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -448,9 +448,12 @@ <h1>Samza Configuration Reference</h1>
                 </tr>
                  <tr>
                     <td class="property" id="job.coordinator.factory">job.coordinator.factory</td>
-                    <td class="default"></td>
+                    <td class="default">org.apache.samza.zk.ZkJobCoordinatorFactory</td>
                     <td class="description">
-			Class to use for job coordination. Currently available values are:
+                        The fully-qualified name of the Java class which determines the factory class which will build the JobCoordinator.
+                        The default configuration value if the property is not present is <code>job.coordinator.factory=org.apache.samza.zk.ZkJobCoordinatorFactory</code>.<br>
```

Comment:
"The default configuration value if the property is not present is <code>job.coordinator.factory=org.apache.samza.zk.ZkJobCoordinatorFactory</code>" -> recommend deleting this line.

Reply:
Done.

Reply:
Done.